index.html

```diff
@@ -196,8 +196,26 @@ <h4>LET'S  CONNECT</h4>
             </ul>
             </div>
            </div>
-              </section> 
+              </section>
+              <section class="contact-area">
+                <div class="contact-me">
+                   <h2 class="contact-title">Contact Me</h2>
+                   <p class="contact-desc">
+                       If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
+                   </p>
+                   
+                           <form action="contact-form" method="post">
```

Comment:
- [ ] [REQUIRED] Good job so far! Your `Form` should use the `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html). You have used `action` and `method` in form but you made some mistake in value given to `action`, Please use formspree to give value to `action`!  `[HINT]` The `action` value should be similar to `https://formspree.io/f/{form_id}`

![image](https://user-images.githubusercontent.com/63915024/192602523-d888d314-b321-4fce-9745-7a385bffcb04.png)